create/faq.md

```diff
@@ -180,20 +180,41 @@ Until our first stable release (v1.0), MeiliSearch minor versions are not compat
 
 ## What are the recommended requirements for hosting a MeiliSearch instance?
 
-**Short answer**: It's impossible to say. We recommend [experimenting with RAM availability and database size](#how-does-hardware-affect-search-speed) to find the hardware + relevancy setup that works best for your dataset.
+**The short answer: we don't know.**
 
-**Long answer**: The disk size and RAM usage of your MeiliSearch database can vary widely. The following factors have a great effect on space and memory use:
+The complexity involved in the indexing process makes it extremely difficult to accurately estimate the size and memory use of a MeiliSearch database, even knowing the size of the `.JSON` files beforehand.
+
+In the future, we hope to provide detailed benchmarks of different use-cases. Until then, read on for more detailed information about optimizing your MeiliSearch instance.
+
+### Memory Usage
+
+There are two things that can cause your memory usage (RAM) to spike:
+
+1. Adding documents
+2. Updating index settings (if index contains documents)
+
+To reduce memory use and indexing time, follow this best practice: **always update index settings before adding your documents**. This avoids unnecessary double-indexing.
+
+:::tip
+MeiliSearch's new engine (release date: before the end of 2021) **will allow you to set a limit on how much RAM MeiliSearch uses**.
```

Comment:
The user will not be able to set the RAM usage once the new search engine is released, it will come later 🙂

Reply:
Didn't know that, thanks @curquiza ! Will delete this sentence.